pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -113,7 +88,7 @@ let
         # https://github.com/neovim/neovim/issues/9413
         wrapProgram $out/bin/nvim \
           --set NVIM_SYSTEM_RPLUGIN_MANIFEST $out/rplugin.vim \
-          --add-flags "-u ${vimUtils.vimrcFile configure}"
+          --add-flags "--cmd \"source ${neovimrcFile}\""
```

Comment:
alternatively I can wrap with ` --set VIMINIT "source vimrc"` to recover the old behavior.